composer.json

```diff
@@ -17,6 +17,7 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
+        "ext-intl": "Required for most features of Zend\\I18n; not included in all default builds of PHP",
```

Comment:
This needs to be a `*` character, and not a sentence, in order to work with Composer. (Only version constraints are allowed as values in the `require` and `require-dev` sections.)

Reply:
Got it. Updating. Don't do that often, and examples showed a sentence.

Reply:
PR updated

Reply:
The "suggest" section is different, to allow detailing _why_ a particular package is recommended, and potentially to provide other information (e.g., what minimum version should be used). `require`, `require-dev`, and `conflicts` all require version constraints.